text_extensions_for_pandas/array/test_span.py

```diff
@@ -455,17 +457,28 @@ def test_addition(self):
 
 class CharSpanArrayIOTests(ArrayTestBase):
 
-    @pytest.mark.skip("Temporarily disabled until Feather support reimplemented")
     def test_feather(self):
         arr = self._make_spans_of_tokens()
```

Comment:
Would you mind modifying these tests so that they write out a `SpanArray` containing spans over two different document texts?

Reply:
I added a separate test for multi-doc.